charts/k8ssandra/values.yaml

```diff
@@ -150,30 +98,41 @@ cass-operator:
   image: jsanda/cass-operator:91205f4d8f1e
 
 kube-prometheus-stack:
-  alertmanager:
-    enabled: false
-
-  kubeStateMetrics:
-    enabled: false
-
-  grafana:
-    enabled: false
-
-  nodeExporter:
-    enabled: false
-
-  kubeletService:
-    enabled: false
-
-  kubeControllerManager:
-    enabled: false
-
-  kubelet:
-    enabled: false
-
-  kubeApiServer:
-    enabled: false
+  # Locks Prometheus operator to this namespace and kube-system
```

Comment:
Why do we need cube-system? Wouldn't this ruin the whole idea of namespacing it?

Reply:
The default service monitors and dashboards that ship with kube-prometheus-stack includes kubelet dashboards. If we are looking at the developer persona I don't want them to see these dashboards in grafana and think something is wrong when they're empty.

Reply:
We should probably remove them in that case (as the point was to make this Cassandra monitoring, not Kubernetes monitoring), otherwise each instance would access the same kube-system and each one would poll everything there, thus increasing the load in the system. If the user wants to monitor their Kubernetes, they should probably use a separate instance for that, or ship Cassandra metrics to an existing Prometheus instance.

Reply:
That's a fair point. I'm 50/50 on either approach. I'll see about the config changes needed to remove those.

Reply:
I tend to agree with @burmanm here.

Reply:
Done with the latest commit.

![Grafana Dashboards](https://user-images.githubusercontent.com/135264/104681888-6e0db100-56c1-11eb-89fb-d6883e3f71d4.png)
![Prometheus Targets](https://user-images.githubusercontent.com/135264/104681934-8382db00-56c1-11eb-9ee0-21204b3c21b6.png)